README.md

```diff
@@ -6,6 +6,10 @@ This template is a great way to get started on a new [Bevy](https://bevyengine.o
 Start with a [basic project structure](#write-your-game) and [CI / CD](#release-your-game) that can deploy to [itch.io](https://itch.io).
 You can [try this template in your web browser!](https://the-bevy-flock.itch.io/bevy-quickstart)
 
+Don't want to read through the whole README? [@ChristopherBiscardi](https://github.com/ChristopherBiscardi) made a video explaining how to use the template from start to finish.
```

Comment:
```suggestion
[@ChristopherBiscardi](https://github.com/ChristopherBiscardi) made a helpful video on how to use this template from start to finish:
```

Reply:
If not this exact wording, my main issue with this line is that the final word wraps to a new line.

Reply:
![image](https://github.com/user-attachments/assets/6512650d-eaeb-4106-bcdd-c143edef6571)

That's on your specific screen settings. I'll change the wording a bit more, but I don't want to spend more time optimizing that.

Reply:
[`9e359d5` (#174)](https://github.com/TheBevyFlock/bevy_quickstart/pull/174/commits/9e359d5093199a49a508caead030b4c6711792d7)